README.md

```diff
@@ -1851,6 +1851,9 @@ The following extractors use this feature:
 #### soundcloud
 * `formats`: Formats to request from the API. Requested values should be in the format of `{protocol}_{extension}` (omitting the bitrate), e.g. `hls_opus,http_aac`. The `*` character functions as a wildcard, e.g. `*_mp3`, and can passed by itself to request all formats. Known protocols include `http`, `hls` and `hls-aes`; known extensions include `aac`, `opus` and `mp3`. Original `download` formats are always extracted. Default is `http_aac,hls_aac,http_opus,hls_opus,http_mp3,hls_mp3`
 
+#### orfon
+* `prefer_segments_playlist`: Some videos are available both as a single file and as a segmented playlist. This option will return the segmented playlist when using `orfon:prefer_segments_playlist`. If you want to not concatenate the segments you will also need to use `--concat-playlist never`.
```

Comment:
```suggestion
* `prefer_segments_playlist`: Prefer a playlist of program segments instead of a single complete video when available. If individual segments are desired, use `--concat-playlist never --extractor-args "orfon:prefer_segments_playlist"`
```